docs/04-jwk.md

```diff
@@ -593,12 +591,12 @@ source: [examples/jwk_fetch_example_test.go](https://github.com/lestrrat-go/jwx/
 
 ## Auto-refreshing remote keys
 
-Sometimes you need to fetch a remote JWK, and use it mltiple times in a long-running process.
-For example, you may act as an itermediary to some other service, and you may need to verify incoming JWT tokens against the tokens in said other service.
+Sometimes you need to fetch a remote JWK, and use it multiple times in a long-running process.
+For example, you may act as an intermediary to some other service, and you may need to verify incoming JWT tokens against the tokens in said other service.
 
 Normally, you should be able to simply fetch the JWK using [`jwk.Fetch()`](https://pkg.go.dev/github.com/lestrrat-go/jwx/v2/jwk#Fetch),
 but keys are usually routinely expired and rotated due to security reasons.
-In such cases you would need to refetch the JWK periodically, which is a pain.
+In such cases you would need to re-fetch the JWK periodically, which is a pain.
```

Comment:
I prefer `refetch`

Reply:
I understand, but "re-fetched" and "re-fetching" is used in other places so how would you like to proceed here?

Reply:
I did, huh. `s/re-fetch/refetch/g` then?

Reply:
Done!